src/test/regress/expected/propagate_statistics.out

```diff
@@ -70,6 +70,20 @@ DROP STATISTICS IF EXISTS s3, sc2.s4, s6;
 DROP STATISTICS s5,s6;
 ERROR:  statistics object "statistics'Test.s6" does not exist
 DROP STATISTICS IF EXISTS s5,s5,s6,s6;
+-- test renaming statistics
+CREATE STATISTICS s6 ON a,b FROM test_stats4;
+DROP STATISTICS s7;
+ERROR:  statistics object "statistics'Test.s7" does not exist
```

Comment:
If `s7` doesn't exist, why do we drop it first?

Reply:
To show the difference before&after rename statement. It will be dropped after rename

Reply:
So, to show "statistics'Test.s7" table didn't exist before? It's an interesting way to do that.

Reply:
In case of someone accidentally creates the same object before